.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
The comment seems unnecessary and doesn't provide any useful information. Consider removing it.